### ISSUE TITLE
TSCONFIG: Error Key: 'pretty' has a value 'true' of an invalid type: boolean

### ISSUE THREAD

**Issue by xbtsw** [alm-tools/alm#298]
If I have `compilerOptions`:`pretty` in `tsconfig.json` then ALM will report that 

```
TSCONFIG: Error Key: 'pretty' has a value 'true' of an invalid type: boolean
```

And stopped parsing any of the project files, since `tsconfig.json` is incorrect...

But the value for pretty is indeed boolean according to schema:
http://json.schemastore.org/tsconfig

**Comment by basarat** [comment]
@xbtsw can you paste the tsconfig.json here please

**Comment by basarat** [comment]
Nevermind. Verified. Fixing :rose:

**Comment by basarat** [comment]
Released with 2.6.0 :rose:

**Comment by xbtsw** [comment]
⚡️ fast
